src/plugins/intel_gpu/src/kernel_selector/cl_kernels/gemm_mmad_int8.cl

```diff
@@ -469,10 +469,6 @@ KERNEL(gemm_mmad_int8)(
     }
 
 #if TILE_NUM == 2
-#if HAS_FUSED_OPS && FUSED_OPS_CAN_USE_PRELOAD
```

Comment:
How does it work in case if `FUSED_OPS_CAN_USE_PRELOAD == true`? I assume that `FUSED_OPS_CALC` will not compile too.

Also, please add unit test for that case.

Reply:
My case is FUSED_OPS_CAN_USE_PRELOAD == true and it works well,  FUSED_OPS_CALC is running in each for-loop, so there's no problem.
Let's me check unit test too.

Reply:
Add unit test